docs/tutorial/typesfuns.rst

```diff
@@ -860,12 +860,12 @@ Dependent Pairs
 
 Dependent pairs allow the type of the second element of a pair to depend
 on the value of the first element. Traditionally, these are referred to
-as “sigma types”:
+as “sigma types”, so in older code you might see Sigma rather than DPair:
```

Comment:
I think the problem with older code is that they have yet to migrate explicit instances of `Sigma` and `MkSigma` across to the new names. If the dependent pair sugar is used then it doesn't matter.

We should really deprecate fully `Sigma` in a couple of releases. But I am in favour of removing the reference to old code. Maybe we should promote the wiki page for 'backwards breaking' changes to the reference guide...